lib/utils/parse/parse_test.go

```diff
@@ -96,67 +101,47 @@ func TestVariable(t *testing.T) {
 		{
 			title: "valid with brackets",
 			in:    `{{internal["foo"]}}`,
-			out: Expression{
-				expr: variable("internal", "foo"),
-			},
+			out:   "foovalue",
 		},
 		{
 			title: "string literal",
 			in:    `foo`,
-			out: Expression{
-				expr: variable(LiteralNamespace, "foo"),
-			},
+			out:   "foo",
 		},
 		{
 			title: "external with no brackets",
 			in:    "{{external.foo}}",
-			out: Expression{
-				expr: variable("external", "foo"),
-			},
+			out:   "foovalue",
 		},
 		{
-			title: "invalid namespaces are allowed",
+			title: "invalid namespace",
 			in:    "{{foo.bar}}",
-			out: Expression{
-				expr: variable("foo", "bar"),
-			},
+			err:   trace.BadParameter(""),
 		},
```

Comment:
Not sure why we ever wanted to allow this, it will always return an error when interpolated